### ISSUE TITLE
import-as-reference fails when (reference) is present but @import at-rule is split over multiple lines

### ISSUE THREAD

**Issue by jaketyler781** [electrovir/stylelint-plugin-import#16]
In our code base, we use prettier to restrict the max line length to 120 characters. Imports that are longer than 120 characters are split into multiple lines like the input provided below. This breaks the `import-as-reference` rule as shown:

Input:
```
@import (reference)
    '@lucid/angular/common/structuredfloatingpanel/structuredfloatingpanelheader/_structuredfloatingpanelheader';
```
Expected behavior: import-as-reference passes
Actual behavior: import-as-reference fails and attempts to fix this line with this solution:
```
@import (reference) (reference)
    '@lucid/angular/common/structuredfloatingpanel/structuredfloatingpanelheader/_structuredfloatingpanelheader';
```
This makes the LESS file unable to compile into CSS.

I can reproduce the issue with one of the tests locally by adding a `\n` in the `accept` parameter, causing the test to fail:
![Screenshot from 2022-08-17 16-32-03](https://user-images.githubusercontent.com/79490987/185255266-d4c0039c-ec41-4eec-8be2-9ce0f9dae648.png)

**Comment by electrovir** [comment]
Thanks for the bug report @jaketyler781! I've reproduced locally with a test, and should be able to get a fix released soon. I'll notify you when I do.

**Comment by electrovir** [comment]
@jaketyler781 this has been fixed in [version `1.0.10`](https://www.npmjs.com/package/stylelint-plugin-import/v/1.0.10), which has just been published. Please re-open this ticket if that new version does not fix the bug you're encountering in your code base!
